spec/models/movie_spec.rb

```diff
@@ -3,5 +3,7 @@
 RSpec.describe Movie, type: :model do
   describe "relationships" do
     it {should belong_to :studio}
+    it {should have_many :movie_actors}
+    it {should have_many(:actors).through(:movie_actors)}
```

Comment:
Terry - I'd love to see you complete the remaining user stories and then please send them along for feedback! you can push them up to this same branch so I can leave comments on this PR